NadekoBot.Core/Modules/CustomReactions/Extensions/Extensions.cs

```diff
@@ -87,7 +87,7 @@ public static Task<string> ResponseWithContextAsync(this CustomReaction cr, IUse
 
         public static async Task<IUserMessage> Send(this CustomReaction cr, IUserMessage ctx, DiscordSocketClient client, bool sanitize)
         {
-            var channel = cr.DmResponse ? await ctx.Author.GetOrCreateDMChannelAsync().ConfigureAwait(false) : ctx.Channel;
+            var channel = cr.DmResponse ? await ctx.Author.CreateDMChannelAsync().ConfigureAwait(false) : ctx.Channel;
```

Comment:
Is there a changelog that describes all of these changes in the discord.net library?

Reply:
https://discordnet.dev/CHANGELOG.html

Reply:
I don't see anything specific about all these big changes. You're sure they're absolutely necessary? They're not really covered in the migration guide either https://discordnet.dev/guides/v2_v3_guide/v2_to_v3_guide.html

Reply:
Yes it is necessary, that method was removed in version 3.1

Reply:
> I don't see anything specific about all these big changes. You're sure they're absolutely necessary? They're not really covered in the migration guide either https://discordnet.dev/guides/v2_v3_guide/v2_to_v3_guide.html

https://discordnet.dev/api/Discord.IUser.html No getorcreatedmchannelasync method here